Game/Room.pde

```diff
@@ -116,10 +131,10 @@ class Room {
     }
   }
 
-
+//-----------------------------------------------------PLEASE MOVE IT TO ANOTHER CLASS-------------------------------

```

Comment:
I agree with moving the logic to logic to the Board class. There is no reason to have it in the Room class.

I´m not sure where to put the display of the text for the items though